rte_mis.install

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @file
+ * Provide hook_install() / hook_uninstall() and hook_update_N() functions.
+ */
+
+use Drupal\Core\Form\FormState;
+
+/**
+ * Implements hook_install().
+ */
+function rte_mis_install() {
+  $form_state = (new FormState())->setValues([]);
+  \Drupal::formBuilder()->submitForm('Drupal\menu_export\Form\MenuImportForm', $form_state);
+}
```

Comment:
Please add a comment explaining the reason of doing this.

Reply:
ok